tools/vanity/tests/test_AddressGenerator.py

```diff
@@ -11,7 +11,8 @@
 class AddressGeneratorTest(unittest.TestCase):
 	@staticmethod
 	def _derive_public_key_at(facade, mnemonic, account_index):
-		coin_id = 1 if 'testnet' == facade.network.name else facade.BIP32_COIN_ID
+		bip32_coin_id = 43
```

Comment:
use facade.bip32_path ?

Reply:
i mean isn't this
`[44, coin_id, account_index, 0, 0]`
same as 
`facade.bip32_path(account_index) ?

Reply:
yup